includes/class-wpsms-gateway.php

```diff
@@ -694,7 +694,7 @@ public static function gateway()
                 'cpsms'    => 'cpsms.dk',
                 'cellsynt' => 'cellsynt',
                 'suresms'  => 'suresms.com',
-                'prosmsdk' => 'prosms.se'
+                'prosmsdk' => 'sms.dk'
```

Comment:
Is this correct? 🤔 @foadadeli

Reply:
Yes, it is the correct name for ProSMS in Denmark. I was confused at first because the names were the same, but I eventually realized that the name was different in Denmark. @mostafasoufi

Reply:
Oh, you right

Thanks